File: mot-selenium/src/main/java/com/dvsa/mot/selenium/datasource/enums/CompanyType.java
package com.dvsa.mot.selenium.datasource.enums;

public enum CompanyType {
    Company("RC", "Company"),
    Partnership("P", "Partnership"),
    SoleTrader("ST","Sole Trader"),
    PublicBody("PA","Public Body");

    private String value;
    private String name;

    private CompanyType(String value, String name) {
        this.name = name;
        this.value = value;
    }

    public String getName() {
        return name;
    }

    public String getValue() {
        return value;
    }
}

File: mot-common-web-module/src/DvsaCommon/Model/CompanyType.php
<?php


namespace DvsaCommon\Model;


use DvsaCommon\Enum\CompanyTypeCode;

class CompanyType
{
    public static function getPossibleCompanyTypes()
    {
        return [
            CompanyTypeCode::PARTNERSHIP,
            CompanyTypeCode::PUBLIC_BODY,
            CompanyTypeCode::SOLE_TRADER,
            CompanyTypeCode::COMPANY,
        ];
    }
}